### DISCUSSION TITLE
Tab Group

### DISCUSSION THREAD

**Discussion by PriyanshuMallick** [mozilla-mobile/fenix#21314] [Ideas and Feature Requests]
I love using Firefox but as I'm a person who opens every link in a new tab it quickly gets very difficult to navigate through multiple tabs. It gets unorganised very quick. 
I also love to use the Brave browser with is based on chromium, It has a feature called 'Tab Group' which groups tab in a folder and shows them on the bottom of the screen for quick access.

**Comment by amedyne** [comment]
Thank you for being a Firefox user @PriyanshuMallick. If you are a[ Firefox Nightly](https://play.google.com/store/apps/details?id=org.mozilla.fenix) user, you can test out initial implementation concepts that we have for Tab grouping. We appreciate if you can provide some feedback.

**Reply by JVariance** [reply]
I really like the idea. It's way more clear 👍🏼

**Reply by suicideboy98** [reply]
i think what he meant by group tabs is [this](https://imgur.com/a/1rk90gE) which is something i also wish firefox had. horizontal tab bar is scrollable when there's many tabs. in brave u need to tap and hold to scroll it which is a waste of time it should be instant like in chrome

**Comment by regs01** [comment]
Groups is a nightmare and terror of Chrome. I hope they will never be pushed by default in Firefox.

**Reply by Bardock88** [reply]
I also don't think this style of group of tabs is very good, it would be nice if it had the option of the tab bar that not even the old Fennec project had, even the reordering of tabs when you hold a tab and put it in the order you want would also help and with sure users would really to like this new feature.
